StudioCore/Editor/TaskManager.cs

```diff
@@ -12,6 +12,25 @@ public class TaskManager
         public static volatile ConcurrentDictionary<string, string> warningList = new ConcurrentDictionary<string, string>();
         private static volatile ConcurrentDictionary<string, (bool, Task)> _liveTasks = new ConcurrentDictionary<string, (bool, Task)>();
         private static int _anonIndex = 0;
+        public static string lastCompletedActionString = "";
```

Comment:
This whole block is adding UI timing specific code to the task manager, which logically should not have anything to do with how the UI is displayed.

Reply:
Will strip out this system.